test/integration/default/apache_conf_spec.rb

```diff
@@ -1,15 +1,17 @@
 # encoding: utf-8
 
-if os.linux?
+# TODO: do not run those tests on docker yet
+unless ENV['DOCKER']
+  if os.linux?
```

Comment:
all of these handlers, including the `os.linux?` check are creating huge diffs.
i would like to avoid these in the future. please us:

```ruby
return if Env['DOCKER']
return unless os.linux?
```